src/web/blocks.rs

```diff
@@ -28,19 +29,20 @@ pub struct BlockShallowDTO {
     timestamp: Timestamp,
     block_hash: SerScaleHex<Hash>,
     transactions: u32,
-    rejected_transactions: u32,
+    // rejected_transactions: u32,
+    signature: SignaturesOf<CommittedBlock>,
 }
-
-impl TryFrom<BlockValue> for BlockShallowDTO {
+impl TryFrom<VersionedCommittedBlock> for BlockShallowDTO {
     type Error = color_eyre::Report;
-
-    fn try_from(block: BlockValue) -> Result<Self> {
+    fn try_from(block: VersionedCommittedBlock) -> Result<Self> {
+        let committed_block = block.into_v1();
         Ok(Self {
-            height: block.header.height.try_into()?,
-            block_hash: block.header.current_block_hash.into(),
-            timestamp: Timestamp::try_from(block.header.timestamp)?,
-            transactions: block.transactions.len().try_into()?,
-            rejected_transactions: block.rejected_transactions.len().try_into()?,
+            height: committed_block.header.height.try_into()?,
+            block_hash: committed_block.hash().into(),
+            timestamp: Timestamp::try_from(committed_block.header.timestamp)?,
+            transactions: committed_block.transactions.len().try_into()?,
+            // rejected_transactions:  committed_block.rejected_transactions.len().try_into()?,
```

Comment:
Just put `0` here if it is unsupported ATM. We should open an issue and put its link here.

Reply:
@horizenight, is this resolved?